src/errors/components/ErrorPage.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { Row, Col } from 'antd';
+import error404 from '../images/404.svg';
+
+class ErrorPage extends Component {
```

Comment:
each error page will be different  like `404Page`.

But all the shared logic should go to a common `ErrorPage` component like this one, where you pass `errorImage`, `name`, `message`